docs/source/faster.md

```diff
@@ -0,0 +1,128 @@
+Accelerating garak runs
+=======================
+
+Sometimes, we want to run garak faster.
+A default run can subsist of 80,000 or more inference requests.
+This may take a long time to execute.
+Thankfully there are a number of options and techniques built into garak and 
+surrounding tooling that are intended to make this workload more accessible.
+This documentation page details how garak runs and what can be done in order to
+speed runs up.
+
+
+
+Target types
+------------
+
+Generators in garak can broadly be thought of as one of two types: local models, and remote endpoints.
+Local models are things like huggingface.Pipeline; remote endpoints are things like openai.OpenAICompatible or nim.NVOpenAIChat.
+
+Local generators consume resources on the same machine as which garak runs.
+This means they only run on machine, and more often than not, it also means that there aren't resources to run more than one instance of a model.
+That has the effect of making runs only possible in serial.
+Local generators also have to some some orchestration of getting models up and running and keeping them running. 
+Since there are so many different kinds of model around, and each of these is often pushing the limits of what's technically and scientifically possible at the time of its release, local generators tend to be a bit prone to mid-run failures, e.g. when a previously-unseen edge case is encountered.
+
+Remote endpoints mean that all this orchestration is sent away to some dedicated service.
+This has a few advantages.
+First, garak doesn't have to do orchestration.
+Second, it can be possible for multiple instances of the target to run in parallel without garak or you having to worry about it.
+Third, the people running the endpoint have often done some quality checking and testing to make sure that the endpoint runs well, reducing the chance of the target crashing weirdly.
+Fourth, because orchestration (i.e. getting models to be loaded, and to run) happens remotely, if there is a failure, the solution can be as simple (from garak's point of view) as re-sending the inference request to the target endpoint. Garak is generally pretty gentle but tenacious when it comes to dealing with endpoint failure - we know that runs can take a while and we want to mitigate the need to "babysit" them, by having garak politely try to recover the target.
```

Comment:
nit: Generally prefer to avoid parens and Latinisms.  Maybe "...because orchestration--loading and serving models--happens remotely,.."  or ", such as loading and serving models,"